VimeoUpload/Descriptor System/DescriptorManagerArchiver.swift

```diff
@@ -47,49 +47,53 @@ class DescriptorManagerArchiver
 
     // MARK: - Initialization
     
-    init(name: String)
+    init(name: String, documentsFolderURL: URL)
```

Comment:
Can we make the initializer failable? It looks that the `archiver` is mandatory for this class.